crates/ursa-rpc-service/src/tests/mod.rs

```diff
@@ -44,16 +43,14 @@ pub fn init() -> InitResult {
     };
     let keypair = Keypair::generate_ed25519();
     let service = UrsaService::new(keypair.clone(), &network_config, Arc::clone(&store))?;
-    let server_address = Multiaddr::try_from("/ip4/0.0.0.0/tcp/0").unwrap();
 
     let provider_engine = ProviderEngine::new(
         keypair,
         Arc::clone(&store),
         get_store(),
         ProviderConfig::default(),
         service.command_sender(),
-        server_address,
-        "/ip4/127.0.0.1/tcp/4069".parse().unwrap(),
+        vec!["/ip4/127.0.0.1/tcp/4069".parse().unwrap()],
```

Comment:
For ease of maintenance this might be better
```suggestion
        ServerConfig::default_addresses(),
```